lib/core/params.js

```
@@ -19,6 +19,23 @@ module.exports = function (Ravel) {
    */
   const defaults = Object.create(null);
 
+  const ENVVAR_PATTERN = /\$([a-zA-Z_][0-9a-zA-Z_]*)/g;
+  /**
+   * Interpolates config values with the values of the environment variables of the process
+   * @private
+   */
+  const interpolateEnvironmentVariables = function (value) {
+    if (typeof value !== 'string') {
+      return value;
+    }
+    const result = value.replace(ENVVAR_PATTERN, function () {
+      const varname = arguments[1];
+      return process.env[varname] || '';
```

Let's throw an exception here. See my comment on the overall PR.

---

:+1: